scripts/preprocessing/03-make_epochs.py

```diff
@@ -16,14 +16,24 @@
 import mne
 from mne.parallel import parallel_func
 from mne_bids import BIDSPath
-
+from autoreject import get_rejection_threshold
 import config
 from config import gen_log_message, on_error, failsafe_run
 
 logger = logging.getLogger('mne-bids-pipeline')
 
 
 @failsafe_run(on_error=on_error)
+
+def _get_global_reject_epochs(raw, events, event_id, tmin, tmax):
+    epochs = mne.Epochs(
+        raw, events, event_id=event_id, tmin=tmin, tmax=tmax,
+        proj=False, baseline=None, reject=None)
+    epochs.load_data()
+    epochs.apply_proj()
+    reject = get_rejection_threshold(epochs, ch_types=['mag', 'grad'])
```

Comment:
why not EEG? We should also pass the decim keyword here.

Reply:
Yes I should add EEG. I got rid of the decim parameter because it is set by default to 1 (https://github.com/mne-tools/mne-bids-pipeline/pull/276#discussion_r606421885).